docs_requirements.txt

```diff
@@ -1,7 +1,8 @@
 sphinx==1.7.9
 sphinx_rtd_theme
 sphinxcontrib-jsonschema
-nbconvert
+nbsphinx
```

Comment:
so just adding this makes sphinx build rst docs from the notebooks that it finds? :) super cool)

Reply:
And loading the plugin from the conf file but otherwise yes